openedx_ledger/models.py

```diff
@@ -287,7 +272,7 @@ class Meta:
             "The globally unique content identifier.  Joinable with ContentMetadata.content_key in enterprise-catalog."
         )
     )
-    reference_id = models.CharField(
+    enterprise_fulfillment_uuid = models.CharField(
```

Comment:
Perhaps take this opportunity to update the `help_text` to make it clear that this field refers to the UUID of a table defined in edx-enterprise.